source/dialogs/bulkconverterdialog.cpp

```diff
@@ -0,0 +1,221 @@
+#include "bulkconverterdialog.h"
```

Comment:
Just noticed this now, but the .h and .cpp files should have the license text at the top (can copy from any other source file)